logistics/scripts/modules/postmanTestHarnessConfiguration.json

```diff
@@ -28,7 +28,7 @@
           "ApiClients": [
             {
               "ApiClientName": "255901",
-              "LocalEducationOrganizations": [255901],
+              "LocalEducationOrganizations": [255901, { "Start": 2147480647, "Count": 3000 }, 9223372036854775000],
```

Comment:
I wish I had seen this line of the diff first -- because it explains everything else.